src/com/palmergames/bukkit/towny/command/TownyAdminCommand.java

```diff
@@ -1592,18 +1603,62 @@ public void purge(String[] split) {
 			} catch (TownyException e) {
 				TownyMessaging.sendErrorMsg(player, e.getMessage());
 			}
-			
-			if (resident != null) {
-				try {
-					ConfirmationHandler.addConfirmation(resident, ConfirmationType.PURGE, days); // It takes the senders town & nation, an admin deleting another town has no confirmation.
-					TownyMessaging.sendConfirmationMessage(player, null, null, null, null);
 
-				} catch (TownyException e) {
-					TownyMessaging.sendErrorMsg(player, e.getMessage());
+			Resident finalResident = resident;
+			String finalDays = days[0];

```

Comment:
Not sure why you've changed the above days to a String[] to make days[0], to create a new finalDays String to do the same logic below as was used previously.